docs/tutorial/tutorial-6.rst

```diff
@@ -1,15 +1,464 @@
+
+      [hello-world] Application updated.
+
+    Rebuild the app:
+
+    .. code-block:: bash
+
+      (beeware-venv) $ briefcase build
+
+      [hello-world] Building AppImage...
+      ...
+      [hello-world] Built linux/Hello_World-0.0.1-x86_64.AppImage
+
+    And finally, run the app:
+
+    .. code-block:: bash
+
+      (beeware-venv) $ briefcase run
+
+      [hello-world] Starting app...
+
+      Traceback (most recent call last):
+        File "/tmp/.mount_Hello_ifthSH/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
+          return _run_code(code, main_globals, None,
+        File "/tmp/.mount_Hello_ifthSH/usr/lib/python3.8/runpy.py", line 87, in _run_code
+          exec(code, run_globals)
+        File "/tmp/.mount_Hello_ifthSH/usr/app/hello_world/__main__.py", line 1, in <module>
+          from hello_world.app import main
+        File "/tmp/.mount_Hello_ifthSH/usr/app/hello_world/app.py", line 8, in <module>
+          import httpx
+      ModuleNotFoundError: No module named 'httpx'
+
+      Unable to start app hello-world.
+
+  .. group-tab:: Windows
+
+    .. code-block:: doscon
+
+    Update the code in the packaged app:
+
+      (beeware-venv)C:\...>briefcase update
+
+      [hello-world] Updating application code...
+      Installing src/hello_world...
+
+      [hello-world] Application updated.
+
+    Rebuild the app:
+
+    .. code-block:: bash
+
+      (beeware-venv)C:\...>briefcase build
+
+      [hello-world] Built windows/msi/Hello World
+
+    And finally, run the app:
+
+    .. code-block:: bash
+
+      (beeware-venv)C:\...>briefcase run
+
+      [hello-world] Starting app...
+
+      Unable to start app hello-world.
+
+Once again, the app has failed to start because ``httpx`` has been installed - but
+why? Haven't we already installed ``httpx``?
+
+We have - but only in the development environment. Your development environment
+is entirely local to your machine - and is only enabled when you explicitly
+activate it. Although Briefcase has a development mode, the main reason you'd
+use Briefcase is to package up your code so you can give it to someone else.
+
+The only way to guarantee that someone else will have a Python environment that
+contains everything it needs is to build a completely isolated Python
+environment. This means there's a completely isolated Python install, and a
+completely isolated set of dependencies. This is what Briefcase is building when
+you run ``briefcase build`` - an isolated Python environment. This also explains
+why ``httpx`` isn't installed - it has been installed in the your *development*
+environment, but not in the packaged app.
+
+So - we need to tell Briefcase that our app has an external dependency.
+
+Updating dependencies
+=====================
+
+In the root directory of your app, there is a file named ``pyproject.toml``.
+This file contains all the app configuration details that you provided when
+you originally ran the ``briefcase new`` wizard.
```

Comment:
```suggestion
you originally ran ``briefcase new``.
```

This isn't referred to as a 'wizard' anywhere else, and the term 'wizard' isn't immediately associated any more to an installation helper/guided experience. So, remove the noun and just talk about what the user previously ran.